R/fontconfig.R

```diff
@@ -64,3 +66,12 @@ fc_match_debug <- function(debug) {
   names(debug) <- "FC_DEBUG"
   debug
 }
+
+# Hack to force Fontconfig to reload debug flag
```

Comment:
This only reloads the dll when `debug` is requested. We need to clean it up otherwise any subsequent call to gdtools functions using fontconfig will produce debug output.

It seems to work well on my machine and I think it's basically how devtools works so this should be well tested. Since I pluck the dll from the namespace info it works with both the installed dll and a `load_all()`'d dll.